src/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3.8
```

Comment:
Suggestion 1: don't specify minor version and you can get more updates as they come
```suggestion
FROM python:3
```